File: dotnet/socketd/org.noear.socketd/exception/SocketdChannelException.cs
namespace org.noear.socketd.exception;

/**
 * 通道异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketdChannelException : SocketdException
{
 public SocketdChannelException(string message) : base(message)
 {

 }

 public SocketdChannelException(String message, Exception cause) : base(message, cause)
 {

 }
}

File: java/socketd/src/main/java/org/noear/socketd/exception/SocketDChannelException.java
package org.noear.socketd.exception;

/**
 * 通道异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketDChannelException extends SocketDException {
    public SocketDChannelException(String message) {
        super(message);
    }

    public SocketDChannelException(String message, Throwable cause) {
        super(message, cause);
    }
}
